tutorials/README.md

```diff
@@ -1,9 +1,22 @@
 ## Tutorials
-This folder contains tutorial notebooks for DIANNA.
-A tutorial is avaiable for several combinations of data modality and explanability method.
-Click one of the links in the table to directly go to a notebook:
+This folder contains DIANNA tutorial notebooks.
 
+Tutorials are avaiable for several combinations of data modality, dataset and explanability method.
+
+Use the links in the tables below to directly go to a tutorial notebook:
+
+<!--- to do: check if LIME is for LeafSNap (Simple Sicnetific) or more. DO we need Images row or each notebook is for 1 dataset only!? -->
+
+### Images
+|Data \ XAI|RISE|LIME|KernelSHAP|
+|:-----|:---|:---|:---|
+|_Images_|[]()|[link](lime_images.ipynb)|[]()|
+|Binary MNIST |[link](rise_mnist.ipynb)| []()| [link](kernelshap_mnist.ipynb)|
+|Simple geometric (circles and triangles) <img width="20" alt="SimpleGeometric Logo" src="https://user-images.githubusercontent.com/3244249/151539027-f2fc3fc0-282a-4993-9680-74ee28bcd360.png">|[]()| []()| [link](kernelshap_geometric_shapes.ipynb)|
+|Simple scientific (Leafsnap30) <img width="20" alt="LeafSnap30 Logo" src="https://user-images.githubusercontent.com/3244249/151539100-dbdfe0f8-485f-45d4-a249-a1f79e970066.png"> []()| []()| []()|
```

Comment:
```suggestion
<!--|Simple scientific (Leafsnap30) <img width="20" alt="LeafSnap30 Logo" src="https://user-images.githubusercontent.com/3244249/151539100-dbdfe0f8-485f-45d4-a249-a1f79e970066.png"> []()| []()| []()|-->
```

Let's comment it out as long as we don't have any notebooks yet, it's a bit weird to have an empty row in the table.

Reply:
I think we have 1 notebook in progress by Yang?